client/blocks/signup-form/index.jsx

```diff
@@ -896,13 +892,48 @@ class SignupForm extends Component {
 						) }
 					</LoggedOutForm>
 
-					<LoggedOutFormLinkItem href={ logInUrl }>
+					<LoggedOutFormLinkItem href={ this.getloginUrl() }>
 						{ this.props.translate( 'Log in with an existing WordPress.com account' ) }
 					</LoggedOutFormLinkItem>
 				</div>
 			);
 		}
 
+		/*
+			AB Test: passwordlessSignup
+
+			`<PasswordlessSignupForm />` is for the `onboarding` flow.
+
+			We are testing whether a passwordless account creation and login improves signup rate in the `onboarding` flow
+		*/
+		if (
+			this.props.flowName === 'onboarding' &&
+			'passwordless' === abtest( 'passwordlessSignup' )
+		) {
+			return (
+				<div className={ classNames( 'signup-form', this.props.className ) }>
+					{ this.getNotice() }
+					<PasswordlessSignupForm
+						stepName={ this.props.stepName }
+						flowName={ this.props.flowName }
+						goToNextStep={ this.props.goToNextStep }
+						userCreationComplete={ this.userCreationComplete() }
+						renderTerms={ this.termsOfServiceLink() }
```

Comment:
Apologies if this PR isn't ready for review yet! Just testing out D31888-code and D31710-code and the call to `this.renderTerms()` in `<PasswordlessSignupForm>` is erroring out and causing a BSOD for me, and I think it's because of this function call here, instead of passing in a reference to the function:

```suggestion
						renderTerms={ this.termsOfServiceLink }
```

Reply:
Thanks for noticing this! I thought I'd fixed it, but, once again, forgot to push that commit 🤣